docs/source/tools.rst

```diff
@@ -92,6 +92,19 @@ Then, add ``pcvl.pdisplay()`` of your circuit.
   :align: center
   :width: 75%
 
+.. tip::
+
+    The outcome of this last command will depend on your environment.
+
+    .. list-table::
+       :header-rows: 1
+       :width: 100%
+
+       * - Text Console
+         - Jupyter Notebook
```

Comment:
Is it normal that you removed the IDE case in this table? I guess it's still something we want to show